src/main/thrift/parquet.thrift

```diff
@@ -243,42 +243,39 @@ struct SizeStatistics {
  */
 struct Statistics {
    /**
-    * DEPRECATED: min and max value of the column. Use min_value and max_value.
+    * max/min: deprecated, used to encoded signed orderable values, ignoring
+    *   the columns ColumnOrder
+    * max_value/min_value: PLAIN encoded values, sans length prefix if varlen
+    * max8/min8: up to 8-bytes:
+    *   FLOAT, DOUBLE: bitcasted to INT32 and INT64, respectively
```

Comment:
I think we might want to be more specific here about values less then 8 bytes are translated into 8 bytes.  In practice it doesn't make a difference for readers but it would be good to limit ambiguity.  I assume we do a normal cast from 1/4 integer byte values to 8 bytes values rather then just embedding them?